java/src/test/java/org/certificatetransparency/ctlog/LogSignatureVerifierTest.java

```diff
@@ -56,13 +32,13 @@
 public class LogSignatureVerifierTest {
   /** Returns a LogSignatureVerifier for the test log with an EC key */
   private LogSignatureVerifier getVerifier() {
-    LogInfo logInfo = LogInfo.fromKeyFile(TEST_LOG_KEY);
+    LogInfo logInfo = LogInfo.fromKeyFile(LogSignatureVerifierTest.class.getResource(TEST_LOG_KEY).getFile());
```

Comment:
I improved the line breaks, some went a bit over 100, but my thought was that it was better that the code pattern repeated between the tests than all lines was below 100 chars.

I don't have any particular opinion about this, just notify me if it should be changed more.